README.md

```diff
@@ -194,34 +151,9 @@ Illustration of how Profiles are added to the Catalog and how they can then be q
 
 ![](/docs/assets/catalog.jpg)
 
-### Profile Subscriptions
+### Profile Installation
 
-Illustration of a basic profile install (aka subscription), using `kubectl` and a hand-written `ProfileSubscription`:
```

Comment:
y'know a less lazy person would have updated the diagrams

Reply:
its no longer relevant though :shrug:

Reply:
I'll update the diagrams in `pctl`, since this repo is no longer related to the installation

Reply:
https://github.com/weaveworks/pctl/pull/87